src/workflows/multiomics/process_batches/main.nf

```diff
@@ -153,13 +158,13 @@ workflow run_wf {
       // Processing of multi-modal multisample MuData files.
       // Performs calculations on samples that have *not* been integrated,
       // and can be considered a "no-integration" workflow.
-
-      output_ch = [dimensionality_reduction_rna, dimensionality_reduction_prot].inject(multimodal_ch){ channel_in, component ->
+      output_ch = [dimensionality_reduction_rna, dimensionality_reduction_scaling_rna, dimensionality_reduction_prot].inject(multimodal_ch){ channel_in, component ->
```

Comment:
why duplicate this dimred component? I think we should just give more or fewer parameters to the dimred_rna component depending on whether `rna_enable_scaling` is set to true. Can we not simply add `enable_scaling` as a parameter to `dimensionality_reduction`?

Reply:
The scaled layers are stored _in addition_ to the non-scaled data. This means that most of the dimensionality reduction components are run twice: once on the scaled data and once on the non-scaled data. There are two options here:

1. Implement this behavior in the `dimensionality_reduction` subworkflow. This means duplicating run most of the component once or twice, depending on the wether or not scaling is enabled.
2. Just run the `dimensionality_reduction` subworkflow with scaling enabled and write to different output slots.

Option number 2 is actually less duplicated code. Indeed at first glance there is duplication on the level of the argument passing to the subworkflow when doing this, but option number 1 is way more duplication on the level of the subworkflow. And a lot more error-prone to make sure that the right arguments end up with the right component in the subworkflow